src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -4191,6 +4221,9 @@ private AccessorDeclarationSyntax ParseAccessorDeclaration(bool isEvent)
                 accessorName = ConvertToKeyword(accessorName);
             }
 
+            // PROTOTYPE: How should ‘field’ and ‘value’ be interpreted in the attributes, modifiers? Those parts have already been parsed.
```

Comment:
we already decided this in ldm, but the answer, afair, is: they are not in scope there.